cmd/cluster-controller/Dockerfile

```diff
@@ -0,0 +1,30 @@
+# Copyright 2018 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Reproducible builder image
+FROM golang:1.10.0 as builder
```

Comment:
The tests are using 1.10.2. Should this be built with 1.10.2 as well? Or can the test use 1.10.0?

Reply:
Looks like travis doesn't support 1.10.0, but it does support 1.10.2. So I switched the build to 1.10.2.